packages/docs/src/locales/en/en.json

```diff
@@ -2930,31 +2930,31 @@
     "description": "Vuestic UI components support keyboard interactions for all mouse-based actions and utilize HTML5 semantic elements where applicable.",
     "keyboardInteractions": {
       "title": "Keyboard accessible",
-      "description": "All the interactive elements of your page e.g. text boxes, buttons etc can be accessible by keyboard. That means you should be able to bring focus on them by using Tab and Shift+Tab keys."
+      "description": "All the interactive elements on your page (e.g. text boxes, buttons, etc.) can be accessed with the keyboard. That means you should be able to focus them by using Tab and Shift+Tab keys."
     }
   },
   "configurationGuide": {
     "title": "Configuration",
-    "description": "We have developed a convenient mechanism that allows you to deeply customize your icons, components, and colors.",
-    "readMore": "Read more",
+    "description": "Vuestic UI comes with a convenient mechanism that allows for a deep icons, components, and colors customization.",
+    "readMore": "More on this topic",
     "colors": {
       "title": "Color themes",
-      "description": "Vuestic UI comes with a set of colors that could be easily extended"
+      "description": "You can easily modify the colors used by Vuestic UI components (and even add custom ones):"
```

Comment:
Usually "could" is used instead of "can" to be more politem up to you.

Reply:
That would sound more like a past tense to me (IMHO). Like if a user had had a capability to modify colors in some previous version, but they don't now.

Would definitely stick to "could" if asking a question though.